src/splat/segtypes/common/code.py

```diff
@@ -283,25 +204,13 @@ def parse_subsegments(self, segment_yaml) -> List[Segment]:
             segment.sibling = base_segments.get(segment.name, None)
 
             if segment.sibling is not None:
-                if self.section_order.index(".text") < self.section_order.index(
-                    ".rodata"
-                ):
-                    if segment.is_rodata():
-                        segment.sibling.rodata_sibling = segment
-                else:
-                    if segment.is_text() and segment.sibling.is_rodata():
-                        segment.rodata_sibling = segment.sibling
-                        segment.sibling.sibling = segment
-
-                if self.section_order.index(".text") < self.section_order.index(
-                    ".data"
-                ):
-                    if segment.is_data():
-                        segment.sibling.data_sibling = segment
-                else:
-                    if segment.is_text() and segment.sibling.is_data():
-                        segment.data_sibling = segment.sibling
-                        segment.sibling.sibling = segment
+                # Make siblings reference between them
```

Comment:
has this been tested with any ps1 decomps (i.e. that have rodata before text?), I'm happy to try if not.

Reply:
>  tested ESA and got the :OK: